python/ctsm/toolchain/gen_mksurfdata_jobscript_single.py

```diff
@@ -7,6 +7,7 @@
 import argparse
 import logging
 
+
```

Comment:
And this is the last script to go over.

Reply:
@samsrabin and I went over this together. One idea to do later would be to separate single and multi into a class that has a base class used by both, and then extends it for each specific script. I'll add that to the issue that talks about this.

Reply:
We also thought of an improvement in the name of the "part1" to include batch in the name. This is easy to do.

Reply:
In issues #2446 and #2439